src/inmanta/data/__init__.py

```diff
@@ -4659,8 +4684,10 @@ async def get_resources_for_version(
                     if no_obj:
                         record = dict(record)
                         record["attributes"] = json.loads(record["attributes"])
+                        parsed_id = resources.Id.parse_id(record["resource_id"])
+                        parsed_id.set_version(record["model"])
+                        record["resource_version_id"] = parsed_id.resource_version_str()
```

Comment:
A comment on why we add `resource_version_id` could be useful here. It's obvious in the context of this PR but it might be less so in a couple of months.

Reply:
I added it to the `__mangle_dict` method